### DISCUSSION TITLE
Startup third cluster node after being shutdown

### DISCUSSION THREAD

**Discussion by simon-appleyard** [ravendb/ravendb#18324] [General]
Hello,

We have a 3 node cluster running on version 6.0.1.

We had to take our third 03 node down for a period of time.

When we restart the 03 node, will the other two 01 and 02 nodes synchronize all data to the 03 node? As the databases on these servers have been the "live" ones. 

We just want to confirm that no data will be synchronized from the 03 node to 01 and 02.

Many Thanks

**Comment by DejanMilicic** [comment]
Once cluster detects node as available again, it will first put him in the rehab state, assign it a mentor node which will sync it with everything that node C missed while it was unavailable.

Can you explain what data could be synchronized from node C while it was down, and why that would be a problem?

**Reply by simon-appleyard** [reply]
Hi,
Thanks for the quick response.
That makes sense.
Think it was just in case any "old" data would be copied back to the other nodes, for some reason. But of course it will sync the newest data to node C which it has missed.
